ncdiff/src/yang/ncdiff/runningconfig.py

```diff
@@ -97,6 +97,11 @@
     (re.compile(r'^ *mpls mldp static '), 0),
     (re.compile(r'^ *router '), 0),
     (re.compile(r'^ *summary-address '), 1),
+    (re.compile(r'^ *mobile-network '), 4),
+    (re.compile(r'^ *ospfv3 neighbor '), 1),
+    (re.compile(r'^ *ospfv3 1 ipv(\d) neighbor'), 1),
+    (re.compile(r'^ *area '), 1),
+    (re.compile(r'^ *ospfv3 1 neighbor '), 1),
```

Comment:
What is this `1` in `ospfv3 1 neighbor`?

Reply:
Hi, i have updated the pass log in above comment and this below log is exact where the issue seen
https://earms-trade.cisco.com/tradeui/logs/logfile?logfile=TaskLog.Task-1&begin=21919020&size=820013&archive=%2Fnobackup%2Fvijviswa%2Fcmt_script%2Fpolaris%2Fbinos%2Flinkfarm%2Fsdks-host%2Fiosxe-pyats%2Fnativesdk%2Fsdk%2Fsysroots%2Fx86_64-xesdk-linux%2Fusr%2Fusers%2Fvijviswa%2Farchive%2F24-06%2Fcli2show_job.2024Jun19_08:53:05.637932.zip&ats=%2Fnobackup%2Fvijviswa%2Fcmt_script%2Fpolaris%2Fbinos%2Flinkfarm%2Fsdks-host%2Fiosxe-pyats%2Fnativesdk%2Fsdk%2Fsysroots%2Fx86_64-xesdk-linux%2Fusr&submitter=vijviswa&from=trade&view=all&atstype=pyATS